python/amici/pysb_import.py

```diff
@@ -9,7 +9,8 @@
     ODEExporter, ODEModel, State, Constant, Parameter, Observable, SigmaY,
     Expression, LogLikelihood, generate_measurement_symbol
 )
-
+# TODO move away from sbml import
+from .sbml_import import noise_distribution_to_cost_function, _get_str_symbol_identifiers
```

Comment:
@FFroehlich : I'd like to move those functions out of `sbml_import`, as they are not sbml-specific. Would not add them to ode_export, as it's rather on the import side (and that file is already too huge). What about introducing something like `import_utils` for general import functions?

Reply:
sounds good!